benchmarks/utils/gen_metrics_json.py

```diff
@@ -0,0 +1,258 @@
+import argparse
```

Comment:
So just trying to understand the end to end story - is this correct?

Right now `benchmark-ab.py` takes configs in json and outputs a bunch of different types of files including csv

This PR 
1. Turn config to YAML
2. Convert to JSON and run `benchmark-ab.py`
3. Get output in csv and images
4. Convert output to json
5. Take json and convert to md file
6. Store md file in a cloud bucket

Reply:
Got it! I think it's worth mentioning this flow in an updated `bechmarks/README.md`

Reply:
I still think it's worth adding these instructions in the README, apologies if I missed it